Swat/Swat.php

```diff
@@ -244,7 +244,7 @@ function dngettext($domain, $messageid1, $messageid2, $n)
 	 * gettext.
 	 *
 	 * @package   Swat
-	 * @copyright 2005-2006 silverorange
+	 * @copyright 2005-2015 silverorange
```

Comment:
I know this is just updating the date, but why does `Swat.php` have multiple copyright blocks?